src/components/network/network-panel.js

```diff
@@ -135,8 +148,21 @@ genotet.NetworkPanel.prototype.setEdgeInfo_ = function(edge, container) {
     .text(edge.source.label);
```

Comment:
But in the renderer, the edge.source is stored as a network.Node. So I need to judge whether edge.source has th attribute label?